Src/BootCamp.Chapter/Computer/Cpu.cs

```diff
@@ -2,5 +2,11 @@
 {
     public class Cpu
     {
+        private string _cpu;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_cpu' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)

Reply:
done

Reply:
done